src/Resources/Locales/en.json

```diff
@@ -116,6 +116,10 @@
         "schemaValidationFailed": {
             "type": "JSON schema validation failed",
             "message": "JSON is invalid. Check syntax matches required schema."
+        },
+        "timeSeriesDatabaseConnectionFetchFailed": {
+            "type": "Time series database connection validation failed",
```

Comment:
should we say "failed" in this case? That makes it sound fatal but we really just mean "we couldn't validate it, keep on doing your thing if you like" Just seems like it's a dead end when I believe it's more of a warning?

Reply:
we used "validation failed" in somewhere else, that is why I used it this way. I can change it to "Time series database connection could not be validated".

Reply:
I'm confused. Why do we have it localized if it's not sured for showing to the user?